cpp/src/Slice/Grammar.y

```diff
@@ -1633,44 +1633,54 @@ ICE_ENUM
     currentUnit->pushContainer(en);
     $$ = en;
 }
-'{' enumerator_list '}'
+'{' enumerators '}'
 {
     currentUnit->popContainer();
     $$ = $1;
 }
 ;
 
+// ----------------------------------------------------------------------
+enumerators
+// ----------------------------------------------------------------------
+: enumerator_list
+| enumerator_list ','
+{
+    $$ = $1;
+}
+| %empty
+{
+    $$ = make_shared<EnumeratorListTok>(); // Empty list
+}
+;
+
```

Comment:
There was no change to the behavior of enums.
This is to allow trailing commas in enums, after switching to left-recursion.